chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -371,7 +371,7 @@ user.subjectName.required=Username ('subject name') is required.
 user.providerKey.invalid=Provided user key is not valid in the database. The user key is valid for up to 3 days from when it is assigned.
 
 #Warnings
-transparencyAttestation.save=Warning: The transparency attestation for the developer is different in the system than in the upload file. To save the updated transparency attestation, as well as all other developer information, please click 'Save as Developer Information' above before proceeding.
+transparencyAttestation.save=The transparency attestation for the developer is different in the system than in the upload file. To save the updated transparency attestation, as well as all other developer information, please click 'Save as Developer Information' above before proceeding.
```

Comment:
I can see one change to a message, but I count three in the Excel file. Are the other two not actually supposed to change?

Reply:
Those two other errors already have the correct text in the code